### ISSUE TITLE
Add support for Unicode character border

### ISSUE THREAD

**Issue by silkfire** [rebus-org/Tababular#6]
Would it be hard to add an option to use a Unicode border instead?

Similar to what this tool has as an option: https://www.tablesgenerator.com/text_tables#

![image](https://user-images.githubusercontent.com/1554615/55790583-f575cb80-5abc-11e9-9cf4-dffd38926d22.png)

**Comment by mookid8000** [comment]
Yeah well, if I remember correctly the code is a little bit convoluted and not really prepared to be extended.

Would be a fun refactoring though, to make it extendable like that.

There's plenty of tests to verify that the current behavior is not broken.

**Comment by silkfire** [comment]
See it as a challenge!
